src/topology/basic.lean

```diff
@@ -382,6 +382,8 @@ begin
     exact h U U_op ⟨_, x_in⟩ },
 end
 
+alias dense_iff_inter_open ↔ dense.inter_open_nonempty _
```

Comment:
Could you explain what the purpose of this is?  Is there a reason not to use the latter form only?

Reply:
This creates an alias for one of two implications in `dense_iff_inter_open`, so that one can write `(h : dense s).inter_open_nonempty (hto : is_open t) (htn : t.nonempty)`

Reply:
The `iff` statement is not useful for dot notation.

Reply:
I see, thanks!